worldedit-core/src/main/java/com/sk89q/worldedit/util/PropertiesConfiguration.java

```diff
@@ -119,6 +120,12 @@ public void load() {
         allowSymlinks = getBool("allow-symbolic-links", allowSymlinks);
         serverSideCUI = getBool("server-side-cui", serverSideCUI);
         extendedYLimit = getBool("extended-y-limit", extendedYLimit);
+        defaultLocaleName = getString("default-locale", defaultLocaleName);
+        if (defaultLocaleName.equals("default")) {
```

Comment:
Shouldn't we extract this if-default-else-tag logic to a function in LocalConfiguration?

Can just do something like `setDefaultLocaleFromName(defaultLocaleName)` then.